lib/api/actions/functions.php

```diff
@@ -91,130 +91,122 @@ function beans_add_smart_action( $hook, $callback, $priority = 10, $args = 1 ) {
 }
 
 /**
- * Modify an action.
+ * Modify one or more of the arguments for the given action, i.e. referenced by its Bean's ID.
  *
- * This function modifies an action registered using {@see beans_add_action()} or
- * {@see beans_add_smart_action()}. Each optional argument must be set to NULL to keep the orginal value.
+ * This function modifies a registered action using {@see beans_add_action()} or
+ * {@see beans_add_smart_action()}. Each optional argument must be set to NULL to keep the original value.
  *
  * The original action can be reset using {@see beans_reset_action()}.
  *
  * @since 1.0.0
+ * @since 1.5.0 Made WPCS compliant and improved action parameter filtering.
```

Comment:
As per discussed, we will not mention WPCS compliancy in the `@since` tag so this may be changed to `@since 1.5.0 Improved action parameter filtering.`.